# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# Filename: package/.../uclibc/pkg_patch/xfsprogs-no_aio_h.patch
# Copyright (C) 2007 The OpenSDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./configure.in.orig	2007-09-10 22:01:14.000000000 -0400
+++ ./configure.in	2007-10-11 22:31:23.000000000 -0400
@@ -39,9 +39,6 @@
 AC_PACKAGE_UTILITIES(xfsprogs)
 AC_MULTILIB($enable_lib64)
 
-AC_PACKAGE_NEED_AIO_H
-AC_PACKAGE_NEED_LIO_LISTIO
-
 AC_PACKAGE_NEED_UUID_H
 AC_PACKAGE_NEED_UUIDCOMPARE
 
--- ./include/builddefs.in.orig	2007-10-12 11:36:20.000000000 -0400
+++ ./include/builddefs.in	2007-10-12 11:36:38.000000000 -0400
@@ -25,7 +25,7 @@
 MALLOCLIB = @malloc_lib@
 LOADERFLAGS = @LDFLAGS@
 
-LIBRT = @librt@
+LIBRT = -lrt
 LIBUUID = @libuuid@
 LIBPTHREAD = @libpthread@
 LIBTERMCAP = @libtermcap@
